src/Mews.Fiscalizations.All/Mews.Fiscalizations.All.csproj

```diff
@@ -10,7 +10,7 @@
     <RepositoryUrl>https://github.com/MewsSystems/fiscalizations</RepositoryUrl>
     <Icon>https://raw.githubusercontent.com/msigut/eet/master/receipt.png</Icon>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageVersion>9.0.0</PackageVersion>
+    <PackageVersion>9.0.1</PackageVersion>
```

Comment:
It's interesting how you don't want to bump this as a major version but you're bumping the version of Basque library. 😄